platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gossip/chatter/protocol/peer/PeerGossipState.java

```diff
@@ -21,49 +21,46 @@
 import com.swirlds.common.sequence.map.StandardSequenceMap;
 import com.swirlds.platform.consensus.GraphGenerations;
 import com.swirlds.platform.gossip.chatter.protocol.messages.ChatterEvent;
-import com.swirlds.platform.gossip.chatter.protocol.messages.ChatterEventDescriptor;
+import com.swirlds.platform.gossip.chatter.protocol.messages.EventDescriptor;
 import org.apache.commons.lang3.ObjectUtils;
 
 /**
  * Keeps track of the state of chatter communication with a peer, including events we are sure the peer knows
  */
 public class PeerGossipState implements Shiftable {
     /** non-ancient events we know the peer knows */
-    private final SequenceMap<ChatterEventDescriptor, ObjectUtils.Null> events;
+    private final SequenceMap<EventDescriptor, ObjectUtils.Null> events;
     /** the maximum generation of all event descriptors received */
     private long maxReceivedDescriptorGeneration;
 
     /**
      * Create a new object for tracking communication with a peer.
      *
-     * @param futureGenerationLimit
-     * 		the maximum number of generations in the future we are willing to accept from
-     * 		the peer
+     * @param futureGenerationLimit the maximum number of generations in the future we are willing to accept from the
+     *                              peer
      */
     public PeerGossipState(final int futureGenerationLimit) {
         events = new StandardSequenceMap<>(
-                GraphGenerations.FIRST_GENERATION, futureGenerationLimit, ChatterEventDescriptor::getGeneration);
+                GraphGenerations.FIRST_GENERATION, futureGenerationLimit, EventDescriptor::getGeneration);
         maxReceivedDescriptorGeneration = GraphGenerations.FIRST_GENERATION;
     }
 
     /**
      * Mark an event represented by this descriptor as known by the peer
      *
-     * @param event
-     * 		the descriptor of the event the peer knows
+     * @param event the descriptor of the event the peer knows
      */
-    public synchronized void setPeerKnows(final ChatterEventDescriptor event) {
+    public synchronized void setPeerKnows(final EventDescriptor event) {
         events.put(event, ObjectUtils.NULL);
     }
 
     /**
      * Query the state about the knowledge of this event
      *
-     * @param event
-     * 		the descriptor of the event being queried
+     * @param event the descriptor of the event being queried
      * @return true if the peer knows this event, false otherwise
      */
-    public synchronized boolean getPeerKnows(final ChatterEventDescriptor event) {
+    public synchronized boolean getPeerKnows(final EventDescriptor event) {
```

Comment:
Null annotations and checks missing

Reply:
I am not going to add those here, since I only changed the type.